parent: !include default.yaml
# parent: !include robots/boxy.yaml
robot_interface:
  base:
    plugin: SendFollowJointTrajectory
    namespace: /whole_body_controller/base/follow_joint_trajectory
    state_topic: /whole_body_controller/base/state
  torso:
    plugin: SendFollowJointTrajectory
    namespace: /whole_body_controller/torso/follow_joint_trajectory
    state_topic: /whole_body_controller/torso/state
  neck:
    plugin: SendFollowJointTrajectory
    namespace: /whole_body_controller/neck/follow_joint_trajectory
    state_topic: /whole_body_controller/neck/state
  left_arm:
    plugin: SendFollowJointTrajectory
    namespace: /whole_body_controller/left_arm/follow_joint_trajectory
    state_topic: /whole_body_controller/left_arm/state
  right_arm:
    plugin: SendFollowJointTrajectory
    namespace: /whole_body_controller/right_arm/follow_joint_trajectory
    state_topic: /whole_body_controller/right_arm/state

